README.md

```diff
@@ -8,6 +8,7 @@ A suite to build, analyze and visualize circuits.
 - [Brayns](#brayns) — Visualizer for large-scale and interactive ray-tracing of neurons.
 - [libsonata](#libsonata) — C++ / Python reader for SONATA circuit files.
 - [BioExplorer](#bioexplorer) - Extract and analyze scientific data for visualization and interactive exploration.
+- [BlueETL](#blueetl) - Multiple simulations analysis tool.
```

Comment:
please put it next to snap.

Reply:
Moved the section close to snap.  Please note that I don't have permissions to merge, so feel free to merge when the changes are good to go.